### ISSUE TITLE
How to split the nli dataset to a dev set and a train set?

### ISSUE THREAD

**Issue by xiami2019** [MatthewCYM/GenSE#1]

**Comment by MatthewCYM** [comment]
Hi @xiami2019,

We randomly sample 250K triplets as the training data for our data synthesizer, and the rest as the validation set.

Thank you!

**Comment by xiami2019** [comment]
Thanks for your reply!
